tekore/_convert.py

```diff
@@ -159,6 +162,8 @@ def from_url(url: str) -> Tuple[str, str]:
     *prefixes, type_, id_ = url.split('/')
     prefix = '/'.join(prefixes)
 
+    id_ = id_.split('?si=')[0]
```

Comment:
Sucks to go back and forth, but the parameters could be in either order. So this should be just `'?'`.

Reply:
And this is debatable, but the empty line isn't probably needed, the split could go in the same block above. Up to you how separate you feel it should be!